next/components/ui/Breadcrumbs/DesktopBreadcrumbs.tsx

```diff
@@ -21,9 +26,13 @@ const DesktopBreadcrumbs = ({ breadcrumbs }: BreadcrumbsProps) => {
           <li className="text-small flex gap-1 font-medium" key={index}>
             <BreadcrumbsChevronIcon className="shrink-0" />
             {breadcrumb.path && !isLast ? (
-              <MLink href={breadcrumb.path} variant="underlined">
-                {breadcrumb.title}
-              </MLink>
+              <Button
+                href={breadcrumb.path}
+                variant="unstyled"
+                className="w-full cursor-pointer outline-none"
```

Comment:
`cursor-pointer` should not be used on random plases, this should be handled in main component.

Never disable `outline-none` if you don't provide other focus style.